pinch/T2A_NOPINCH.DATA

```diff
@@ -0,0 +1,276 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+
+-- Test model for keyword PINCH and grids where there is a gaps between layer K and K+1.
+-- The model includes a Grid with dimension 1x1x3 where here is a gap of 1.0 m between 
+-- layer 2 and 3.
+
+-- The model includes 1 well completed in cell 1,1,1. Summary vectors 
+-- BPR (block pressure) will reveal whether there is communication or not 
+-- from layer 2 to 3. 
+
+-- Since PINCH keyword is not used in this deck, there should be communication
+-- across layer 2 and 3. Hence Bottom layer should not be depleted.
```

Comment:
Either I'm reading it incorrectly or this statement is self-contradictory.  Is there a "no" missing in "there should be communication" here, or should the second sentence read "... should be depleted"?  By the way, the `PINCH.md` file says there should be a non-zero `TRANZ` value for this case (T2A_NOPINCH).

Reply:
Thanks @bska , the inconsistency in the header of T2A_NOPINCH has been updated.  There should be communion and bottom layer should be depleted.